tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -119,6 +119,17 @@ sub run {
         # but not implemented in file_content_replace() yet poo#120690
         $escaped_token =~ s/\&/\\\&/g;
         set_var("HANA_TOKEN", $escaped_token);
+
+        if (get_var('PTF_INSTALL')) {
+            my $ptf_files = qesap_az_list_container_files(
+                storage => get_required_var('HANA_ACCOUNT'),
+                container => (split("/", get_required_var('HANA_CONTAINER')))[0],
+                token => get_required_var('HANA_TOKEN'),
```

Comment:
is it the escaped one right?

Reply:
Here, yes, BUT even if we provide the un-escaped token, it will still work (`qesap_az_list_container_files` un-escapes the token internally, if it is escaped)